docs/src/pages/components/radio-buttons/radio-buttons.md

```diff
@@ -3,7 +3,7 @@ title: Radio buttons React component
 components: Radio, RadioGroup, FormControl, FormLabel, FormControlLabel
 ---
 
-# Radio buttons
+# Radio
```

Comment:
If we have one page per component, I think that we should name it after the component: radio.

Reply:
What about we can it "Radio Group"?

Reply:
> I think that it's interesting to name it after the React component name

I follow the logic, but "radio buttons" is the canonical name.

I believe the reason the components are called Radio and RadioGroup rather than RadioButton and RadioButtonGroup is simply for brevity.

Also, following that logic:

App Bar -> AppBar
Bottom Navigation -> BottomNavigation
Click Away Listener -> ClickAwayListener
CSS Baseline -> CssBaseline
Date / Time -> DatePicker / TimePicker
Expansion Panel -> ExpansionPanel
Grid List -> GridList
Material Icons  -> AcUnit / AccessAlarm / AccessAlarms / AccessTimes / [...] 😄
Speed Dial -> SpeedDial
Textarea Autosize -> TextareaAutosize
Text Field -> TextField
ToggleButton -> ToggleButton
Transfer List -> TransferList

Reply:
I think that the component page title should be close to the name we use in the imports.
If you think this change is better, why not. I don't have a strong preference.

Reply:
> If you think this change is better

No, I don't.

Reply:
Ok, reverted.